### ISSUE TITLE
Python script for reading parquet

### ISSUE THREAD

**Issue by elan-dcc** [elan-dcc/data-loading#3]
Perhaps we also need the loading scripts for parquet files for the one or two Python users (they exist!) Something for you, @Lischip ?

**Comment by Lischip** [comment]
Just for parquet files opposed to sav files?

**Comment by elan-dcc** [comment]
Yes, same as for the R stuff. We have a few Python users who might not realize the value using the parquet files.

**Comment by jvelumc** [comment]
Perhaps you can also include a short snippet like this in one of the examples:
```
library(arrow)
library(dplyr)

# read in only rows for the patients in our studypop, with date after 2020-01-01 onwards, 
# with medication code starting with "B01"
open_dataset("parquet.file") |> 
  filter(
    rinpersoon %in% df_studypop$rinpersoon,
    datum >= "2020-01-01",
    grepl("^B01", dATC)
  ) |>
  select(rinpersoon, datum, dATC, dosis) |> 
  collect()
```
This way only the columns & rows that satisfy the criteria are read into memory. Saves a lot of time if you dont have to read in the whole file and subset afterwards.

**Comment by harcel** [comment]
Merged PR with example code to main on 20240603
